package.json

```diff
@@ -30,12 +30,12 @@
     "topo": "2.x.x"
   },
   "peerDependencies": {
-    "hapi": ">=10 <17"
+    "hapi": "16.x.x"
```

Comment:
This was correct as-is– we support hapi v10 thru 16.

Reply:
werd! will update. thanks, dude!